release/create_release_archives.sh

```diff
@@ -140,28 +140,26 @@ find tools -type f -not -name "githubContrib*" -not -name ".*" -exec "${CP}" --p
 popd
 
 for unwanted_manifest in \
+    istio-auth-non-mcp.yaml \
+    istio-auth-sds.yaml \
+    istio-non-mcp.yaml \
+    istio.yaml \
+    istio-auth.yaml \
+    istio-auth-mcp.yaml \
+    istio-auth-multicluster.yaml \
+    istio-mcp.yaml \
     istio-one-namespace.yaml \
     istio-one-namespace-auth.yaml \
     istio-one-namespace-trust-domain.yaml \
-    istio-multicluster.yaml \
-    istio-auth-multicluster.yaml \
-    istio.yaml \
-    addons/zipkin.yaml \
-    istio-auth.yaml \
-    istio-remote.yaml; do
+    istio-remote.yaml \
+    istio-minimal.yaml \
```

Comment:
this is in our doc, pls don't put it in unwanted_manifest.

I think istio-remote.yaml may also be used?  @ymesika

Reply:
I believe that values-istio-minimal is used, not this. I can double check tomorrow. But either way we maybe want to keep it so helm isn't needed

Reply:
@linsun We don't use the `istio-remote.yaml` anymore nor asking users to.
We have a values file that replaces it as it doesn't make sense to have a static file due to dynamic arguments needed for the remote configuration.

Reply:
thanks!   yes please keep the istio-minimal.yaml.

Reply:
`istio-miniminal.yaml` is not used in the [current docs](https://istio.io/docs/setup/kubernetes/minimal-install/) but its probably useful to keep around anyways.